Fix for: Using dialog to remove table header when tbody is empty throws an error (#1460)
ckeditor/ckeditor-dev: CHANGES.md

## CKEditor 4.11.4

Fixed Issues:

> * [#1397](https://github.com/ckeditor/ckeditor-dev/issues/1397): Fixed: Using dialog to remove headers from [table](https://ckeditor.com/cke4/addon/table) with one row causes JavaScript Error.

Author: A bit more corrections:
```suggestion
* [#1397](https://github.com/ckeditor/ckeditor-dev/issues/1397): Fixed: Using dialog to remove headers from the [table](https://ckeditor.com/cke4/addon/table) with one headers row only throws an error.
```
